front/src/Components/Menu/GlobalMessagesSubMenu.svelte

```diff
@@ -19,12 +19,12 @@
         uploadAudioActive = true;
     }
 
-    function send() {
+    async function send(): Promise<void> {
```

Comment:
Maybe I don't understand but, why we need to add `async` in this function if we never use `await`?

Reply:
`handleSendText.sendTextMessage` returns "void".
`handleSendAudio.sendAudioMessage` returns "Promise<void>"

The async keyword will automatically turn the "void" into a resolved promise returning "void".
Otherwise, I should explicitly write:

```
return Promise.resolve();
```